### DISCUSSION TITLE
Great work. Using this for a new project.

### DISCUSSION THREAD

**Discussion by uklooney** [mrcodetastic/ESP32-HUB75-MatrixPanel-DMA#188] [Show and tell]
Hi,
Thanks for all the work done on this. It has got me started on a audio visualisation project...

![example_matrix_animation1](https://user-images.githubusercontent.com/4552572/134411936-7a78bcd8-3f0b-4158-94ce-c7ecf8680e67.gif)

The code is a bit slap dash at the moment, but output is looking good (pictures don't do justice). If anyone wants a play, the WIP can be found here...
https://github.com/uklooney/AuroraDrop

Thanks All
:)

**Comment by mrcodetastic** [comment]
This is super cool. I'm going to try this out - giving me Winamp visualiser vibes.

Edit: Added a 'Cool uses of this library' section to the readme of this library now and included this.

**Reply by uklooney** [reply]
Thanks. Made a few improvements already. :)

**Reply by uklooney** [reply]
Some Updates.

Now working with a pair of 64x32 HUB75D matrix panels.

![example_matrix_animation1](https://user-images.githubusercontent.com/4552572/134411163-7eed342a-f806-48df-8284-c8c2a170b4af.gif)
![example_matrix_diagnostics1](https://user-images.githubusercontent.com/4552572/134822397-68405460-cfea-4bf5-96b5-e20ee566a729.png)
![example_matrix_ttgo](https://user-images.githubusercontent.com/4552572/138756428-43d786c5-0d22-42ad-aecd-2dd7b31ed488.png)

[![AuroraDrop](http://img.youtube.com/vi/6lSb6BLZaRQ/0.jpg)](http://www.youtube.com/watch?v=6lSb6BLZaRQ "AuroraDrop 0.0.3")

https://youtu.be/6lSb6BLZaRQ


[![AuroraDrop](http://img.youtube.com/vi/wxiZug6A2b8/0.jpg)](http://www.youtube.com/watch?v=wxiZug6A2b8 "AuroraDrop 0.0.12")

https://youtu.be/wxiZug6A2b8


## Example Visualisations

![bitmap1](https://user-images.githubusercontent.com/4552572/137008263-2cd191cb-fc51-445f-8496-6b9dbc1141d4.png) ![bitmap2](https://user-images.githubusercontent.com/4552572/137008280-2f5a7f18-abf7-4f87-937f-abaded38be8e.png) ![bitmap3](https://user-images.githubusercontent.com/4552572/137008303-60a0098d-e878-4fbd-9695-de55e5871269.png) ![bitmap4b](https://user-images.githubusercontent.com/4552572/137008672-e3eb3f7c-4648-4aa2-9c18-655a2434ecf0.png) ![bitmap5](https://user-images.githubusercontent.com/4552572/137008329-1d3b0771-f201-4ce3-95eb-2ec01f6bdd49.png) ![bitmap6](https://user-images.githubusercontent.com/4552572/137008351-81d03100-3593-4c5c-9b21-47538324329a.png) ![bitmap7](https://user-images.githubusercontent.com/4552572/137008366-5f6928c5-a48a-4004-a101-cd3d620e5407.png) ![bitmap8](https://user-images.githubusercontent.com/4552572/137008378-f9b676e0-64e0-4fb9-9871-30986dce088b.png) ![bitmap9](https://user-images.githubusercontent.com/4552572/137008395-985d0761-5988-4116-b357-d0211af1e97c.png) ![bitmap10](https://user-images.githubusercontent.com/4552572/137008412-79c7ce00-66a7-4961-95a7-424caecddbec.png) ![bitmap11](https://user-images.githubusercontent.com/4552572/137008425-967617eb-f75f-4568-bb84-c1349df86457.png)
![bitmap12](https://user-images.githubusercontent.com/4552572/137008461-89f3ff07-3f4a-471f-a605-05168c0964dd.png) 


## Web Interface

![web_bitmap1](https://user-images.githubusercontent.com/4552572/137007058-e84c9805-5442-4d5b-83c0-64f012611f9f.png)

![web_bitmap3](https://user-images.githubusercontent.com/4552572/137807502-aee10806-0850-44fd-ae82-9f524399069c.png)

**Reply by mrcodetastic** [reply]
That's just awesome.
